docs/source/release_notes.rst

```diff
@@ -9,6 +9,7 @@ Future Release
     * Fixes
     * Changes
         * Add autonormalize as an add-on library (:pr:`1840`)
+        * Add Date to Holiday Transform Primitive (:pr: `1846`)
```

Comment:
Minor detail here, but I would stylize this as DateToHoliday - use the class name without spaces.  See line 24 below for an example.  Also be sure to add your username to the list of contributors to the release on line 19.

Reply:
Fixed